.github/workflows/build-extra.yml

```diff
@@ -31,20 +31,26 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
     - uses: actions/checkout@v2
-    - name: configure
-      run: CC=clang-11 ./configure --enable-fatal-warnings
-    - name: make
-      run: make
+    - name: install dependencies
+      run: sudo apt-get install ninja-build
+    - name: Install meson
+      run: pip install --pre meson
```

Comment:
why do you install meson with pip, instead of apt?

Reply:
Because apt have an "old" version and I want to have a workflow which uses the latest release(-[candidate](https://nibblestew.blogspot.com/2021/10/a-call-for-more-downstream-testing-of.html)) of meson (and one using the oldest supported by us).

In the long run it is IMHO best to have an own workflow for meson compatibility which runs [once per week](https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#scheduled-events) and on changes of meson.build files. But for now I didn't wanted to struggle to much with it.